packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -285,13 +371,53 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
         className: chatMessageSlotClassNames.actionMenu,
         styles: resolvedStyles.actionMenu,
       }),
+      overrideProps: {
+        id: actionMenuId.current,
+      },
     });
 
-    if (!actionMenuElement) {
-      return actionMenuElement;
+    const content = actionMenuElement ? <Ref innerRef={actionsMenuRef}>{actionMenuElement}</Ref> : actionMenuElement;
+
+    return inlineActionMenu || !content ? content : <PortalInner>{content}</PortalInner>;
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent) => {
```

Comment:
We should have the tabbable reaction menu in our examples and E2E tests, I think we don't have anything on it currently

Reply:
Yeah make sense. I will update e2e test with tabable element in actionMenu

Reply:
@ling1726 do you mean to able use "TAB" key inside the actionMenu?
I been looking into it and figure out the whole actionMenu is toolbar. And inside the toolbar there should be just right or left arrow keys used to navigate. Not TAB really. 
If there is some implementation that inside the toolbar is TAB key working, it is custom and a "hacky" way.
So I would not rather use it in our actionMenu examples...